cluster/gce/addons/konnectivity-agent/konnectivity-agent-rbac.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: konnectivity-agent
+  namespace: kube-system
+  labels:
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Is it a right place to create a server binding permissions on agent file?

Reply:
Thanks for the catch. This should be bundled with the konnectivity server creation.